StyleCop.Analyzers/StyleCop.Analyzers.Test/DocumentationRules/SA1627UnitTests.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+namespace StyleCop.Analyzers.Test.DocumentationRules
+{
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Analyzers.DocumentationRules;
+    using Microsoft.CodeAnalysis.Diagnostics;
+    using TestHelper;
+    using Xunit;
+
+    /// <summary>
+    /// This class contains unit tests for <see cref="SA1627DocumentationTextMustNotBeEmpty"/>.
+    /// </summary>
+    public class SA1627UnitTests : DiagnosticVerifier
+    {
+        public static IEnumerable<object[]> Elements
+        {
+            get
+            {
+                yield return new[] { "remarks" };
+                yield return new[] { "example" };
+                yield return new[] { "permission" };
+                yield return new[] { "exception" };
+            }
+        }
+
+        /// <summary>
+        /// Checks an element with a blank value gives an error.
+        /// </summary>
+        /// <param name="element">Element to check</param>
+        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+        [Theory]
+        [MemberData(nameof(Elements))]
+        public async Task TestMemberWithBlankElementAsync(string element)
+        {
+            var testCode = @"
+/// <summary>
+/// Foo
```

Comment:
:bulb: I've been trying to move away from `Foo` in test strings since it's flagged as a spelling error. :stuck_out_tongue_winking_eye: